docker-images/toolbox/resources/docker-compose.yml

```diff
@@ -37,6 +37,7 @@ kubelet:
   volumes_from: [ "kubelet-volumes", "kubelet-pki" ]
   environment:
     USE_CNI:
+    WEAVE_VERSION:
```

Comment:
This also needs to be set in `toolbox.dockerfile`, otherwise it'll get reset and latest CNI plugin is used in any case, which is not desirable.